src/resources/clients.ts

```diff
@@ -411,6 +411,27 @@ export class Clients extends Resource<{realm?: string}> {
     queryParamKeys: ['scope', 'userId'],
   });
 
+  public evaluateGenerateIdToken = this.makeRequest<
+  {id: string; scope: string; userId: string},
+  object
+  >({
+    method: 'GET',
+    path: '/{id}/evaluate-scopes/generate-example-id-token',
+    urlParamKeys: ['id'],
+    queryParamKeys: ['scope', 'userId'],
+  });
+
+  public evaluateGenerateUserInfo = this.makeRequest<
+  {id: string; scope: string; userId: string},
+  object
```

Comment:
This method seems to return a type of [`IdToken`](https://www.keycloak.org/docs-api/16.1/rest-api/index.html#_idtoken), let's make this reflect that.